assets/scss/flex.scss

```diff
@@ -220,6 +212,15 @@ Class                   | Variants
 
 }
 
+/*
+Alignment variants are also available using `align-items` (see also: <a class="link" target="_blank" href="https://css-tricks.com/almanac/properties/a/align-items/">css tricks flexbox demo</a>)
+
+Class                   | Variants
+----------------------- | -------------------------------
+`flex--alignTop`        | aligns content to `flex-start`
```

Comment:
we can leave these classes and merge properties with existing `align--[variant]` classes later